Tools/jit/mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+files = Tools/jit
+pretty = True
+python_version = 3.11
```

Comment:
Not ready for 3.12 yet?

Reply:
> Not ready for 3.12 yet?

Mypy doesn't support 3.12 (such features as 695 PEP) at the moment.

Reply:
This pins the lower version, which is 3.11